src/backend/electron/renderer/menuBarData.ts

```diff
@@ -43,7 +43,7 @@ export const useElectronMenuBarData = (
             engineIcons.value[engineInfo.uuid],
           disableWhenUiLocked: false,
           subMenu: removeNullableAndBoolean([
-            engineInfo.path != "" && {
+            Boolean(engineInfo.path) && {
```

Comment:
falsyだったらという意図を持たせたいという意図もあったりしますかね？ 👀 

これは防御的プログラミングをするのではなく、そもそも型的にそういうのが来ないようにプログラミングするのを徹底するのが良い気がしました･･･！！
防御的プログラミングはきりがないので。。

どちらかというと、このバグは防御的プログラミングをしてなかったたからではなく、`""`に意味をもたせたのが良くなかったということかなぁと。

---

PRのタイトルを叶えるなら、こうするのが良い気がしました！！
```suggestion
            engineInfo.isDefault && {
```
これが変数の意味と意図が一致している状態で、なるべくこういうコードになるよう目指せればよいのかな～と思いました！！

偉そうなこと言ってますが、僕はこの`engineInfo.path != ""`の意図を忘れていたので、`デフォルトエンジンを意図してる`と翻訳してくださったの助かりました。。。
おかげで良い提案ができました！！

Reply:
自分もisDefaultの方が良さそうな気がします。

Reply:
将来的にデフォルトエンジンがvvppでインストールされるようになる可能性を考えると`engineInfo.path != undefined`の方がいいような気がします。

Reply:
vvppだった場合も開けないほうが良いのかなと思いました！
ちょっとフォルダを開く機能をなぜ入れたのか覚えていないのですが、「消せるようにするため」だったらこれかなーと。

Reply:
確かspeaker_infoをいじるCoeiroinkのような、エンジンのフォルダに対して色々やりたいと言うエンジンに対応するためのはずです。

Reply:
なるほどです！
であればまあ、isDefaultで良さそうな気がしますね！